### PR TITLE
emscripten 1.34.11

### PR DESCRIPTION
**Pull request by jiegec**

### CODE REVIEW

**Comment by jiegec** [comment]
Oh, that Yosemite machine in travis is so slow... Passed in jenkins.

File: Library/Formula/emscripten.rb [hunk 1]
```diff
@@ -3,27 +3,20 @@ class Emscripten < Formula
...
     end
   end
 
-  bottle do
-    sha256 "fd4d9efb734935996a761128dfaf8a1389d2745ec2bd01d54fab03c2e582bacd" => :el_capitan
-    sha256 "99abf785baf4616d5c1a92ffc8540b07faa03a291e4213a960780b4829d4e118" => :yosemite
-    sha256 "46794913b20d07af986f34b27e435a26209b426094f5eb596e44052aeefee997" => :mavericks
-    sha256 "ff99dde33e1becbb604f6aa3b251b37c494c4da2fecb94916d25e50a6975f8d2" => :mountain_lion
-  end
-
```
Comment (bfontaine) [inline Library/Formula/emscripten.rb]: Please don’t remove this block; it’ll be updated by our bot.

Comment (jiegec) [inline Library/Formula/emscripten.rb]: Ok.

Comment (jiegec) [inline Library/Formula/emscripten.rb]: Done.

**Comment by jiegec** [comment]
Weird time out error. Maybe another Travis CI issue.

**Comment by bfontaine** [comment]
Don’t worry about the Travis build here :wink:

**Comment by bfontaine** [comment]
Thank you for your contribution to Homebrew @jiegec; we appreciate it! :tada:

**Pull request closed without merging**